crates/matrix-sdk/src/utils.rs

```diff
@@ -218,9 +218,32 @@ pub fn is_room_alias_format_valid(alias: String) -> bool {
     has_valid_format && is_lowercase && RoomAliasId::parse(alias).is_ok()
 }
 
+/// Given a pair of optional `body` and `formatted_body` parameters,
+/// return a formatted body:
+///
+/// - If a `formatted_body` exists, return it.
+/// - If it doesn't exist but there is a `body`, parse it as markdown and return
+///   the result.
+/// - Return `None` if there are no `body` or `formatted_body` parameters.
```

Comment:
Think this comment could be more concise: "return the formatted body if available, or interpret the body as markdown, if provided."